### DISCUSSION TITLE
How to activate a scenario?

### DISCUSSION THREAD

**Discussion by anmamun71** [DHI/mikepluspy#52] [Q&A]
I was able to query and access the information inside the model that I need but I want to be able to see it for specific scenarios that are in my model setup of MIKE+.

I am able to see the "active simulation" but that is not the actual scenario active which I believe is the base scenario.

**Comment by ryan-kipawa** [comment]
Hi! We'll look into this starting by adding a notebook, I made an issue for it here #53
